README.md

```diff
@@ -220,6 +222,9 @@ Initializes the server views manager where:
               where `callback` has the signature `function(err, compiled)` where `compiled` is a
               function with signature `function(context, options, callback)` and `callback` has the
               signature `function(err, rendered)`.
+            - `prepare(config)` - initializes additional engine state.
```

Comment:
compile mode was added for backwards compatibility issues. I would just do it async style and be done.